GliaWidgetsTests/CallVisualizer/VideoCall/VideoCallTests.swift

```diff
@@ -0,0 +1,90 @@
+import XCTest
+@testable import GliaWidgets
+
+final class VideoCallTests: XCTestCase {
+    func test_vc_deinit() {
+        weak var weakViewController: CallVisualizer.VideoCallViewController?
+        autoreleasepool {
+            let viewController = CallVisualizer.VideoCallViewController(
+                props: .init(videoCallViewProps: .mock),
+                environment: .mock
+            )
+            weakViewController = viewController
+        }
+        XCTAssertNil(weakViewController, "VisitorCodeViewController not deinitilized")
+    }
+    func test_end_screen_sharing() {
+        let viewModel: CallVisualizer.VideoCallViewModel = .mock()
+        viewModel.environment.screenShareHandler.status.value = .started
+
+        let headerProps: Header.Props = .init(
```

Comment:
Isn't there a way to get the `.mock()` one and just change the one you are interested in? I think in this case it is the `endScreenshareButton`. That way the test is a lot shorter and easier to read. This applies to all the mocks that you are creating for subsequent tests as well.

Reply:
fixed [3c6d82a](https://github.com/salemove/ios-sdk-widgets/pull/496/commits/3c6d82aa1dae1b906a78bffc8a43018b3ad75965)